website/common/script/content/spells.js

```diff
@@ -382,6 +388,7 @@ spells.special = {
     cast (user) {
       user.stats.buffs.seafoam = false;
       user.stats.gp -= 5;
+      user.pinnedItems = user.pinnedItems.filter(item => !item.path.includes('sand'));
```

Comment:
can we use the exact path here? So if in future we have a new item with `sand` in its path we don't risk removing it by accident. Same for the other places

Reply:
Sure, 
I'm not sure that understand you clearly... How can we resolve it here without using 'sand' string to filter items?

Reply:
Now I understand, we should use full path of item here. Will be done @paglias

Reply:
Done